### DISCUSSION TITLE
change faker's renovate schedule?

### DISCUSSION THREAD

**Discussion by HaveF** [rclement/datasette-dashboards#203] [General]
https://github.com/rclement/datasette-dashboards/pull/200#issuecomment-1820050650

the faker package [changed too too frequent](https://github.com/joke2k/faker/blob/master/CHANGELOG.md)...do we need add some renovate config? (because I'm watching this repo, too many mail for this package)

https://docs.renovatebot.com/key-concepts/scheduling/#schedule-when-to-update-specific-dependencies

**Comment by rclement** [comment]
Hi @HaveF , thanks for the suggestion! I have little internet connectivity currently and little time for OSS projects but I'll keep that in mind. Or if you want to submit a PR that would be very welcome 🙂

I'll turn this conversation into an issue

**Comment by rclement** [comment]
Closing in favor of issue #206
